### PR TITLE
This will mean that we can put status icons on the status page

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by beckermr** [comment]
Can you explain this @CJ-Wright?

**Comment by CJ-Wright** [comment]
One of my issues with the status page is that we don't display any icons about if a PR is mergable or not, so it is difficult to prioritize. Adding this info means that the status page can display the mergability of the PR next to the feedstock link or however we render that in JS.

File: conda_forge_tick/status_report.py [hunk 1]
```diff
@@ -166,10 +167,15 @@ def graph_migrator_status(
             out["in-pr"].add(node)
             fc = "#31688e"
             fntc = "white"
+            pr_status = pr_json["PR"]['mergeable_state']
+            if pr_status == 'clean':
+                status_icon = ' ✓'
+            else:
+                status_icon = ' ❎'
```
Comment (beckermr) [inline conda_forge_tick/status_report.py:174]: should we crib github here? Why is the "x" not red?

Comment (CJ-Wright) [reply conda_forge_tick/status_report.py:174]: Red and Green aren't colorblind friendly, and I'm not certain how to turn individual elements of text colors in graphviz

Comment (beckermr) [reply conda_forge_tick/status_report.py]: ahhh right - good call!

**Pull request merged**